src/test/java/games/strategy/thread/ThreadPoolTest.java

```diff
@@ -112,8 +112,8 @@ public void run() {
     public void run() {
       synchronized (this) {
         try {
-          wait(10);
-        } catch (final InterruptedException ie) {
+          Thread.sleep(10L);
```

Comment:
we've a thread util class out there to help avoid the need to catch interrupted exception, a small simplification possible here.  NBD though either way :+1:

Reply:
Forgot about `ThreadUtil#sleep()`.  Thanks for the reminder!